core/modules/startup/keyboard.js

```diff
@@ -65,7 +65,7 @@ exports.startup = function() {
 				}
 			}
 			if(!hasChanged) {
-				var shortcutConfigTiddlers = $tw.wiki.filterTiddlers("[all[shadows+tiddlers]prefix[$:/config/shortcuts/]]");
+				var shortcutConfigTiddlers = $tw.wiki.filterTiddlers("[all[shadows+tiddlers]prefix[$:/config/shortcuts]]");
 				for(i = 0; i < shortcutConfigTiddlers.length; i++) {
```

Comment:
IMO trailing slash is needed! otherwise prefix handling will fail for `$:/config/shortcutsXXX/` which would be a "false positive"

Reply:
Just found: `$:/config/shortcuts-not-mac/` .. how does the existing mechanism handle it?

Reply:
in the control panel you can set shortcuts for linux only. that tiddler will be named $:/config/shortcuts-linux/something

Reply:
yea saw it. .. replied to early :)

Reply:
but you're right speaking about false-positives because this will update also if only a editor-shortcut changes

Reply:
I'm not handling the different mac, linux, not-mac, not-linux ... config tiddlers, just all with prefix `$:/config/shortcuts` - I was expecting that you'll want that to be changed. The `$:/core/modules/keyboard.js` has that handling, I can use parts of it I guess

For distinguishing between editor and global-keyboard shortcuts I'm lacking ideas